airflow/providers/amazon/aws/operators/glacier.py

```diff
@@ -54,3 +54,55 @@ def __init__(
     def execute(self, context: Context):
         hook = GlacierHook(aws_conn_id=self.aws_conn_id)
         return hook.retrieve_inventory(vault_name=self.vault_name)
+
+
+class GlacierUploadArchiveOperator(BaseOperator):
+    """
+    This operation adds an archive to a vault
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GlacierUploadArchiveOperator`
+
+    :param account_id: The AWS account ID of the account that owns the vault.
+        You can either specify an AWS account ID or optionally a single `-` (hyphen),
+        in which case Amazon S3 Glacier uses the AWS account ID associated with
+        the credentials used to sign the request
+    :param vault_name: The name of the vault
+    :param archive: A bytes or seekable file-like object. The data to upload.
+    :param description: The description of the archive you are uploading
```

Comment:
What is the reason for giving new names to the names boto3 choose?
It's OK to do it I'm just interested in the reason.
Specifically the function has parameter `archiveDescription`  In the operator we set `archive` and `description` but archive is actually body in boto3 and description is `archiveDescription` isn't that a bit confusing?
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glacier.html#Glacier.Client.upload_archive

Reply:
@eladkal No strong opinion.
`archive` is more sound like a file object than `body` and `description` is small and easy to remember than   `archiveDescription` so `archive` and `description` for me look more natural than `body` and `archiveDescription` but happy to change it. Please let me know WDYT

Reply:
@o-nikolas @ferruzzi @vincbeck I'll leave the naming choices for AWS
If its OK from your side I'll accept it as is

Reply:
I tend to agree with @eladkal, I like to reuse boto3 names because, to me, it makes everything easier. But I guess it is only my personal opinion

Reply:
I don't usually like to chime in with "me too", but I do the same.  I keep the names the same whenever possible to make it easier for folks who are used to using boto directly.  So I guess we're going from "personal opinion" to "unwritten rule" here.

That said, I do see why these were changed, those were some misleading names and the new ones do make sense.  I could see making an exception to the "rule" here.

Reply:
Yeah, so far we've tried to stick to boto names (but _not_ the camel case they use). But I'd say it's more of a best practice than a hard rule (though we should start codifying these unwritten rules and best practices for the Amazon provider package in an RST file, we've had this in the back of our mind for a while now).
In this case if you think the boto names are wildly misleading, I'd be happy to approve that.

Reply:
Thanks guys for the feedback. Rename these two parameters. 
```
description -> archive_description
archive -> body
```

Reply:
LGTM